{"name": "Тест на невозможность создать двух курьеров с одинаковым логином", "status": "passed", "start": 1730743275500, "stop": 1730743275877, "uuid": "33ba1bb0-4327-428f-8519-cc1859a4ffdd", "historyId": "91650e0927ed7aafd9ef9ed4a7f8915a", "testCaseId": "91650e0927ed7aafd9ef9ed4a7f8915a", "fullName": "tests.couriers.test_create_couriers.TestCreateCourier#test_create_two_couriers_with_same_login_error", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_create_couriers"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "R-0156"}, {"name": "thread", "value": "85947-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_create_couriers"}]}